### DISCUSSION TITLE
[Python] <sdk for zks-lite>

### DISCUSSION THREAD

**Discussion by ManutdGTA** [zkSync-Community-Hub/zksync-developers#100] [SDKs] [answered]
### Environment

Mainnet

### Acknowledgement

- [X] I understand that issues are expected with the Go SDK as they are being updated.

### Issue Description

Is there any sdk for zks-lite?

### Expected Behavior

I can use zks-lite sdk to sign messages

### Code Example

Is there any sdk for zks-lite?

**Comment by uF4No** [comment, marked as answer]
For zkSync Lite you can use the SDKs here: https://docs.zksync.io/api/
For zkSync Era, you have to use the SDKs from here: https://era.zksync.io/docs/api/

**Reply by ManutdGTA** [reply]
Thank you, bro. I have been able to use SDK and successfully Mint NFT. But now I have encountered problems on Swap Token. There are two Orders in the document. For example, I want to swap ETH to USDC now, but I don’t know how to get the second order。
here is the doc,by the way ,submitter is wrong, I changed 'wallet'
```
orderA = await walletA.get_order('USDT', 'ETH', Fraction(1500, 1), RatioType.token, Decimal('10.0'))
orderB = await walletB.get_order('ETH', 'USDT', Fraction(1, 1200), RatioType.token, Decimal('0.007'))
tx = await submitter.swap((orderA, orderB), 'ETH')
```

**Comment by bradfordbradv** [comment]
At the moment, there isn't a dedicated SDK specifically for zkSync Lite. However, you can interact with zkSync Lite by utilizing the existing zkSync SDKs available for various programming languages, such as JavaScript, Python, or Rust. These SDKs provide functionality for interacting with zkSync contracts, signing messages, and performing transactions on zkSync's Layer 2 network. You can adapt these SDKs to work with zkSync Lite by configuring them to communicate with zkSync Lite endpoints.

**Comment by platactiomeb** [comment]
Environment: Mainnet
Acknowledgement: Issues may occur with the Go SDK due to updates.
Issue Description: Inquiring about the availability of an SDK for zks-lite.
Expected Behavior: Ability to use zks-lite SDK to sign messages.

**Comment by lupanncoofonpowell** [comment]
Currently, there is no specific SDK available for zkSync Lite. However, you can directly interact with the zkSync Lite API using standard HTTP requests or integrate it into existing SDKs as needed.

**Comment by spatounmas** [comment]
Unfortunately, as of now, there is no official SDK specifically tailored for zks-lite. However, you can explore building a custom solution using available libraries and tools provided by the zkSync ecosystem.

**Comment by dianocsemintrey** [comment]
The user is inquiring about the availability of an SDK for zks-lite on the zkSync Mainnet. Specifically, the user wants to know if there is an SDK available to sign messages using zks-lite. No code example or repo link is provided.
